superset-frontend/src/explore/components/controls/MapViewControl/MapViewPopoverContent.tsx

```diff
@@ -28,64 +27,13 @@ export const StyledButtonContainer = styled.div`
   margin: 8px;
 `;
 
-export const StyledCloseButton = styled(Button)`
```

Comment:
I really don't understand why someone re-implemented the style of a button here (?)